main_test.go

```diff
@@ -71,19 +68,27 @@ func TestValidToken(t *testing.T) {
 	client, serverURL := startForTesting(t)
 
 	// Add a relation to make the permission valid.
-	_, err := client.Write(context.Background(), &v0.WriteRequest{
-		Updates: []*v0.RelationTupleUpdate{
-			{
-				Operation: v0.RelationTupleUpdate_CREATE,
-				Tuple: &v0.RelationTuple{
-					ObjectAndRelation: &v0.ObjectAndRelation{Namespace: "test/dashboard", ObjectId: "foobar", Relation: "viewer"},
-					User: &v0.User{UserOneof: &v0.User_Userset{
-						Userset: &v0.ObjectAndRelation{Namespace: "test/token", ObjectId: "sometoken", Relation: "..."},
-					}},
+	_, err := client.WriteRelationships(
+		context.Background(),
+		&v1.WriteRelationshipsRequest{
+			Updates: []*v1.RelationshipUpdate{
+				{
+					Operation: v1.RelationshipUpdate_OPERATION_CREATE,
+					Relationship: &v1.Relationship{
+						Resource: &v1.ObjectReference{
+							ObjectType: "test/dashboard",
+							ObjectId:   "foobar",
+						},
+						Relation: "viewer",
```

Comment:
ditto

Reply:
This is actually writing the relationship. It's correct to be viewer -- the test is actually using view to check.